src/Controller/TestController.php

```diff
@@ -1,23 +1,21 @@
 <?php
+
 /**
- * Created by PhpStorm.
- * User: lisa.backer
- * Date: 8/25/15
- * Time: 8:04 AM
+ * @file
+ * Contains \Drupal\acquia_lift\Controller\TestController.
  */
 
 namespace Drupal\acquia_lift\Controller;
 
-
 use Drupal\Core\Controller\ControllerBase;
 
 class TestController extends ControllerBase {
   public function test() {
-    $webapi = \Drupal::service('acquia_lift_data_connector');
-    $segments = $webapi->getSegments();
-    $build['segments'] = array(
+    $data_api = \Drupal::service('acquia_lift.service.api.data_api');
+    $segments = $data_api->getSegments();
+    $build['segments'] = [
       '#markup' => print_r($segments, TRUE),
-    );
+    ];
     return $build;
   }
 } 
```

Comment:
See my other comment - I think this file should be deleted as it is not in use and doesn't really provide use beyond development.